### ISSUE TITLE
Legend title is at the bottom

### ISSUE THREAD

**Issue by jerometonnelier** [danielgindi/Charts#4950]
Hi!

oddly, the legend title is at the bottom of the legends. I searched the Android doc, but I can't find a way to display it at the top.
Is it a bug, a feature, something missing ? 

Thanks! (and Thanks for this awesome library, it's really gorgeous ;-) )

**Comment by odanu** [comment]
`legend: Legend` is a property of `ChartBaseView`. It contains the necessary properties for position configuration. 
<img width="442" alt="Screenshot 2022-11-01 at 17 24 56" src="https://user-images.githubusercontent.com/1110408/199271036-cd09ec98-7e6a-4d21-b9fe-15a85dc8a997.png">

**Comment by JeromeTonnelierOgury** [comment]
Yes, theses options allow us to position the whole legend object relatively to the chart.
My issue is with the title from the DataSet that is at the bottom of the legend list.
